azadi_pkg.core

```diff
@@ -0,0 +1,29 @@
+CAPI=2:
+name : ::azadi_pkg:1.0
+filesets:
+  risc:
+    depend:
+      - lowrisc:prim:prim_pkg
+      - pulp-platform.org::fpnew_pkg
+      - lowrisc:ibex:ibex_pkg
+    files:
+      #- src/vendor/lowrisc_ibex/rtl/ibex_tracer_pkg.sv
```

Comment:
Same goes with this file, remove commented out lines.

Reply:
Sure.